app/models/payment.rb

```diff
@@ -91,20 +109,22 @@ def customer_attrs
         card_debit:       card.debit,
```

Comment:
Should we store something like `payment_method_type` that'd correspond to whether the previous transaction or subscription was a card or a PayPal payment? Or a `payment_data` json field that'd contain card data if a card was used, and PayPal information if PayPal was used (although I don't like the idea of complicating to that degree).